osu.Game/Rulesets/Mods/ModAutoplay.cs

```diff
@@ -25,6 +25,8 @@ public abstract class ModAutoplay : Mod, IApplicableFailOverride, ICreateReplayD
         public bool RestartOnFail => false;
 
         public override bool UserPlayable => false;
+        public override bool ValidForMultiplayer => false;
+        public override bool ValidForMultiplayerAsFreeMod => false;
```

Comment:
This may be one down from using separate boolean properties (having to specify each and every one to make sense), maybe we can enforce validity of those via `Debug.Assert` in `Mod` constructor? not sure if that's pleasant at all though.